assets/nginx.conf

```diff
@@ -1,9 +1,22 @@
 server {
   listen 5005;
+  
+  
+  location /collect {
+    proxy_pass_request_headers on;
+    proxy_pass_request_body on;
+    proxy_pass http://alloy:8027/collect;
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $scheme;
+  }
```

Comment:
**question (blocking):** Is my understanding correct, that this allows anyone, without authentication, to upload logs via this endpoint?